### ISSUE TITLE
[semantic-issue] Add semantics of sklearn DecisionTreeClassifier

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#173]
**Description**
Invokes the constructor to instantiate a DecisionTreeClassifier object.
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
from sklearn.tree import DecisionTreeClassifier

tree = DecisionTreeClassifier()
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html

**Note (optional)**
Returns an object of DecisionTreeClassifier type.
